src/style.js

```diff
@@ -28,11 +28,8 @@ const addHoverStyle = function(element, props, children) {
   cleanedProps['data-hover-style-id'] = hoverId
   const cssString =
     `[data-hover-style-id="${hoverId}"]:hover {
-    ${toCss(props.hoverStyle, {
-      value: function(value) {
-        return value + ' !important'
-      }
-    })}}`
+    ${toStyle(props.hoverStyle).split(';').join(' !important;')} !important
```

Comment:
Seems fragile. Is it really safe to assume that a semicolon will never appear within a CSS expression?

Reply:
Good point. It would only be able to occur outside of a string. There must be a regex that'll deal with that.

Reply:
You know, since this is only for applying hover styles, if we really need a semicolon in a hover style (which I doubt), we can just put the html escape sequence, which is `&#59;`

Reply:
Suppose a new developer joins this team, starts exploring, and calls this function with a semicolon in the hover style for whatever reason. They would not see the expected behavior and be inclined to blame themselves. Would their mistake be obvious as well as the remedy?

Reply:
Yes. Their semicolon could only appear in a string, and it would be replaced by ` !important;`, so they'd see it immediately, and only in a `:hover` style.